invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -68,9 +68,13 @@ class DashboardTabs extends Component {
         rootElement.dataset[_camelCase(pane.configDataAttribute)]
       );
       return {
-        menuItem: pane.label,
+        menuItem: (
+          <Menu.Item key={index} className="border-orange-bottom">
```

Comment:
This shouldn't create any problems, as no new items should be dynamically removed or added. Right?